lib/components/form/advanced-settings-panel.tsx

```diff
@@ -177,6 +178,15 @@ const AdvancedSettingsPanel = ({
     )
   )
 
+  const handleModeButtonToggle = setModeButton(
+    enabledModeButtons,
+    onSettingsUpdate(setQueryParam)
+  )
+
+  const handleAllSubmodesDisabled = (modeButton: ModeButtonDefinition) => {
```

Comment:
Could we potentially add a setTimeout onto this to match the animation timing when the panel collapses? Right now you uncheck the last one, all the checkboxes check themselves, and then the panel collapses. It's looking a little more like a bug than a feature.

Reply:
I agree it could be better. I believe this will need to be handled in otp-ui. I can open a PR there

Reply:
here is that PR: https://github.com/opentripplanner/otp-ui/pull/791